### DISCUSSION TITLE
Ability to see list of embedded fonts

### DISCUSSION THREAD

**Discussion by Mr-Personality** [sumatrapdfreader/sumatrapdf#3294] [Ideas]
There's only one thing Sumatra PDF can't do that I miss from Acrobat, and that's simply to see the list of used fonts.

Are there any plans to add this information, such as to the File Info dialog? I wouldn't have thought it would be too difficult to add?

Then again, the program doesn't seem to have been updated for several years now? Is development stopped?

**Comment by kjk** [comment]
It's already implemented in latest 3.4.6 version`Ctrl + D`, `Get Font Info`.

https://www.sumatrapdfreader.org/download-free-pdf-viewer

3.4.6 was released mid last year (https://www.sumatrapdfreader.org/docs/Version-history) and there are changes weekly https://github.com/sumatrapdfreader/sumatrapdf/commits/master

**Reply by Mr-Personality** [reply]
Oh, sorry, I was mistaken. I went to your homepage and right there at the top there's a big image that says "v3.3.3" so I assumed this was the latest version!

**Comment by GitHubRulesOK** [comment]
@kjk it might help if there was some wrapping of long entries, this one ran out mid C
![image](https://user-images.githubusercontent.com/12754913/220527611-0ce54729-7722-4ced-b945-f79e4a11116d.png)
